libqfieldsync/project_checker.py

```diff
@@ -128,6 +128,11 @@ def __init__(self, project: QgsProject) -> None:
                 "fn": self.check_layer_package_prevention,
                 "scope": None,
             },
+            {
+                "type": Feedback.Level.WARNING,
```

Comment:
```suggestion
                "type": Feedback.Level.ERROR,
```
Since the problem is like shooting yourself in the feet, we better disallow this to happen.